frontend/sheets/src/states/reducers.js

```diff
@@ -41,7 +41,7 @@ import {
 const defaultSessionState = {
     isFetching: false,
     username: null,
-    isLoggedIn: false,
+    isLoggedIn: true,
```

Comment:
should this be false?

Reply:
yah, was testing, ill change it in a sec.